### PR TITLE
Home page and creating wallet card-components

### PR DESCRIPTION
**Pull request by jradness**
- I created a sass directory with some core sass partials so styles can be more scalable in the future.
- I created a page directory to hold page components and put the home component there.
- I created a components directory to hold reusable components across the project and added the wallet card components there.
- I re-organized some of the styles that were stored in the global styles and put them in the respective component styles.

Clicking the buttons through the creating wallet process swaps out the wallet card components.
I made the service public in the constructor so I could access the booleans in the html. There's probably a better way to set that up, just wanted to point it out.

The UI supports desktop right now. It has not been styled for mobile devices yet.

### CODE REVIEW

**Comment by CrackTheCode016** [comment]
Nice job Jason!  You beat me to it haha!

File: src/app/components/wallet-created/wallet-created.component.scss [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  }
+  .checkbox-group {
+    padding: 5px 0 30px;
+  }
+  form {
+    display: flex;
+    flex-direction: column;
+  }
+}
+
```
Comment (spentak) [inline src/app/components/wallet-created/wallet-created.component.scss]: inconsistent newlines

File: src/app/components/wallet-start/wallet-start.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
```
Comment (spentak) [inline src/app/components/wallet-start/wallet-start.component.spec.ts]: remove newline

File: src/app/services/wallet-card-vm.service.spec.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { TestBed, inject } from '@angular/core/testing';
+
```
Comment (spentak) [inline src/app/services/wallet-card-vm.service.spec.ts]: remove newline

File: src/app/components/components.module.ts [hunk 4]
```diff
@@ -0,0 +1,26 @@
+// Core
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
```
Comment (stuhoffman) [inline src/app/components/components.module.ts]: Remove spaces within imports

**Pull request merged**